symplyphysics/laws/kinematic/maximum_height_from_velocity.py

```diff
@@ -0,0 +1,25 @@
+from sympy import (Eq, solve)
```

Comment:
Move law and test to `dynamics` category.

Reply:
moved